src/plugins/intel_cpu/src/scratch_pad.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common/memory.hpp"
+#include <memory>
+#include "cpu_memory.h"
+#include "dnnl_extension_utils.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class ScratchPad {
+    dnnl::engine eng;
+    Memory m_curScratchpadMem;
```

Comment:
NIT. One class member has `m_` prefix bug the other does not.

Reply:
changed